content/docs/other-guides/usage-reporting.md

```diff
@@ -4,30 +4,39 @@ description = "Opting in or out of anonymous usage data reported from Kubeflow"
 weight = 40
 +++
 
-When enabled, Kubeflow will report **anonymous** usage data using [spartakus](https://github.com/kubernetes-incubator/spartakus), Kubernetes' reporting tool. Spartakus **does not report any personal information**. See [here](https://github.com/kubernetes-incubator/spartakus) for more detail.
-This is entirely voluntary and you can opt out by doing the following:
+When enabled, Kubeflow will report **anonymous** usage data using 
+[Spartakus](https://github.com/kubernetes-incubator/spartakus), Kubernetes' 
+reporting tool. Spartakus does not report any personal information. 
+See the [Spartakus docs](https://github.com/kubernetes-incubator/spartakus) for 
+more detail. 
+
+Allowing usage reporting is entirely voluntary.
+**Reporting usage data is one of the most significant contributions you can make 
+to Kubeflow; so please consider allowing the reporting of usage data.** 
+The data helps the Kubeflow community to improve the project and helps the many 
+companies working on Kubeflow justify continued investment.
+
+**To opt out of usage reporting,** run the following command:
 
 ```bash
-# Delete any existing deployments of spartakus
 kubectl delete -n ${NAMESPACE} deploy spartakus-volunteer
 ```
 
-**To disable usage reporting** you need to delete spartakus component. 
-This command completely deletes any spartakus deployment, while the above 
-command only restarts spartakus with reportUsage set to `false`
+**To disable usage reporting,** you need to delete the Spartakus component. 
```

Comment:
I think you can delete this paragraph; its redundant with the previous paragraph which describes removing spartakus by issuing a delete.

We no longer have a reportUsage flag; Instead people remove spartakus from app.yaml as described below.

Reply:
Done.